packages/manager/modules/pci/src/projects/new/payment/payment.html

```diff
@@ -67,6 +67,13 @@
                         <span data-translate="pci_project_new_payment_use_credits"
                               data-translate-values="{ href: $ctrl.getStateLink('credits') }">
                         </span>
+                        <p class="black-friday mt-2">
```

Comment:
Maybe you could have created a reusable component as you duplicate 
Also how come you have a custom style for the new project but not for credits ?

Reply:
credit is a child element of project that's why the style is only specified once